src/natcap/invest/delineateit/delineateit.py

```diff
@@ -571,13 +560,16 @@ def snap_points_to_nearest_stream(points_vector_path, stream_raster_path_band,
             continue
 
         point = shapely.wkb.loads(source_geometry.ExportToWkb())
+        if geom_name == 'MULTIPOINT':
+            point = point.geoms[0]  # take the first geometry of the multipoint
```

Comment:
What about making this comment say something like, "we already checked & know there is only one point"?

Reply:
Sure! Patched in da09cb3e